modules/virt-creating-vm-template.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-template_{context}"]
+= Creating a virtual machine from a template
+
+You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console.
+
+.Procedure
+
+. Click *Virtualization* -> *Catalog* in the side menu.
+
+. Click *Boot source available* to filter templates with boot sources.
++
+The catalog displays the default templates. Click *All Items* to view all available templates for your filters.
+
+. Click a template tile to view its details.
+
+. Click *Quick create VirtualMachine* to create a VM from the template.
```

Comment:
I admittedly haven't checked for myself in the UI - should this be `*Quick create VirtualMachine*` even though this is not the Quick Create VM module? I'm having trouble differentiating between this module and modules/virt-quick-creating-vm.adoc